src/common/BaseAnnotator.ts

```diff
@@ -63,9 +61,8 @@ export default class BaseAnnotator extends EventEmitter {
             this.rootEl.classList.remove('ba');
         }
 
+        eventManager.removeAllListeners();
```

Comment:
I think it should be safe, since we're destroying the annotator at this point. But the docs themselves say this is not a best practice, so I suppose we can move to a more explicit pattern of removal